lib/howl/inspect.moon

```diff
@@ -139,7 +139,12 @@ show_popup = (editor, inspections, pos) ->
     keep_alive: true,
   }
 
-display_inspections = (editor) ->
+display_inspections = ->
+  timer.on_idle (config.display_inspections_delay / 1000), display_inspections
```

Comment:
Isn't this a perpetual infinite loop?

Instead, why not trigger in 'cursor-changed' and from on_idle?